docs/toc.yml

```diff
@@ -501,11 +501,13 @@
   - name: Get Started
     href: csharp/getting-started/
   - name: Tutorials
-    href: csharp/tutorials/index.md
     items:
-    - name: Introduction to C#
-      href: csharp/tutorials/intro-to-csharp/index.md
+    - name: Index of tutorials
```

Comment:
Shouldn't we stick with overview in these cases?

Reply:
I think you're right.